src/Markdig/Syntax/QuoteBlock.cs

```diff
@@ -12,6 +14,15 @@ namespace Markdig.Syntax
     /// <seealso cref="ContainerBlock" />
     public class QuoteBlock : ContainerBlock
     {
+        // ws? >? ws? block|inline
+        public class QuoteLine
```

Comment:
Avoid nested class. Move this above as a QuoteBlockLine or similar.

Reply:
done